docs/src/main/asciidoc/datasource.adoc

```diff
@@ -559,44 +549,28 @@ XA transactions on reactive datasources are not supported at the moment.
 
 [NOTE]
 ====
-If your transaction involves other, non-datasource resources,
-keep in mind *those* resources might not support XA transactions,
-or might require additional configuration.
+If your transaction involves other non-datasource resources, keep in mind these resources might not support XA transactions, or might require additional configuration.
```

Comment:
Again feel free to ignore, but just in case this was unintended...

Reminder that commas are the difference between "Let's eat, Grandpa!" and "Let's eat Grandpa!" ...

In my mind:

1. "other, non-datasource resources" = "other resources that happen to not be datasources"
2. "other non-datasource resources" = "other resources that are not datasource, **like this one**"

So this change seems to change the meaning of the sentences.

```suggestion
If your transaction involves other, non-datasource resources, keep in mind these resources might not support XA transactions, or might require additional configuration.
```

Reply:
Hey, @yrodiere, great catch again. You are right here; I didn't have the right context. That's why we are asking for reviews.

To explain it better and to make it even clearer, can we go with:

If your transaction involves other resources, which are specifically non-datasource resources, be aware that these resources might not support XA transactions or might require additional configuration.

Reply:
That sounds a bit clunky to me and repeats the word 'resources'... I'd go either with Yoann's suggestion, or

If your transaction involves other types of resources (non-datasource),...

Reply:
Then I'd just go with Yoann's suggestion :) that's accurate and clear too, and doesn't repeat 'resources' too many times.

Reply:
OK, I am currently for accepting @yrodiere suggestion, but wouldn't it be possible to phrase it like this?

If your transaction involves non-datasource resources, be aware that they might not support XA transactions or might require additional configuration.

Reply:
I think that's fine